### PR TITLE
New: Added mcq support for _canShowCorrectness

### PR DESCRIPTION
**Pull request by oliverfoster**
part of https://github.com/adaptlearning/adapt_framework/issues/3597

### New
* Added support for _canShowCorrectness

### CODE REVIEW

File: less/plugins/adapt-contrib-mcq/mcq.less [hunk 1]
```diff
@@ -45,3 +45,19 @@
...
+    left: inherit;
+    right: @item-padding;
+
+    .dir-rtl & {
+      right: inherit;
+      left: @item-padding;
+    }
+  }
+
+  .mcq__text {
```
Comment (kirsty-hames) [inline less/plugins/adapt-contrib-mcq/mcq.less:60]: `.mcq__text` should be `.mcq-item__text`

**Review by kirsty-hames — CHANGES_REQUESTED** [review]
Looking good thanks. Please see comment regarding selector name change.

**Review by joe-allen-89 — APPROVED** [review]
👍

**Review by simondate — APPROVED** [review]

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

**Pull request merged**